api/src/shared/config.js

```diff
@@ -328,6 +328,12 @@ const configuration = (function () {
       url: _removeTrailingSlashFromUrl(process.env.CYPRESS_LCMS_API_URL || process.env.LCMS_API_URL || ''),
       apiKey: process.env.CYPRESS_LCMS_API_KEY || process.env.LCMS_API_KEY,
     },
+    llm: {
+      temporaryStorage: {
+        expirationDelaySeconds: parseInt(process.env.LLM_CHAT_TEMPORARY_STORAGE_EXP_DELAY_SECONDS, 10) || 43200,
+      },
+      getConfigurationUrl: _removeTrailingSlashFromUrl(process.env.LLM_API_GET_CONFIGURATIONS_URL || ''),
```

Comment:
Il faut ajouter les nouvelles variables dans le schéma de la config (ligne 91).

Pour le délai, l’utilisation de la dépendance `ms` est pas mal, car elle permet de mettre des valeurs comme `"12h"` ou `"7d"`

```suggestion
        expirationDelaySeconds: parseInt(process.env.LLM_CHAT_TEMPORARY_STORAGE_EXP_DELAY_SECONDS, 10) ?? 43200,
      },
      getConfigurationUrl: _removeTrailingSlashFromUrl(process.env.LLM_API_GET_CONFIGURATIONS_URL ?? ''),
```